training/general/src/environments/pytorch-1.11-ubuntu20.04-py38-cuda11-gpu/spec.yaml

```
@@ -18,3 +18,5 @@ tags:
   OS: Ubuntu20.04
   Training: ""
   Preview: ""
+  Deprecated: ""
+  Delete-on: "2023-03-20"

```

Let's use `DeleteOn` instead, for consistency with the other tags.
```suggestion
  DeleteOn: "2023-03-20"
```